src/match_openssh.c

```diff
@@ -17,11 +17,52 @@
 
 #define OPENSSH_LINE_MAX 8192	/* from openssh SSH_MAX_PUBKEY_BYTES */
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
and probably not libressl

Reply:
Updated to check for LIBRESSL_VERSION_NUMBER